PurchasesCoreSwift/Public/Purchases.swift

```diff
@@ -0,0 +1,1275 @@
+                                                  subscriberAttributesManager: subscriberAttributesManager)
+        let productsRequestFactory = ProductsRequestFactory()
+        let productsManager = ProductsManager(productsRequestFactory: productsRequestFactory)
+        let offeringsManager = OfferingsManager(deviceCache: deviceCache,
+                                                operationDispatcher: operationDispatcher,
+                                                systemInfo: systemInfo,
+                                                backend: backend,
+                                                offeringsFactory: offeringsFactory,
+                                                productsManager: productsManager)
+        let purchasesOrchestrator = PurchasesOrchestrator(productsManager: productsManager,
+                                                          storeKitWrapper: storeKitWrapper,
+                                                          systemInfo: systemInfo,
+                                                          subscriberAttributesManager: subscriberAttributesManager,
+                                                          operationDispatcher: operationDispatcher,
+                                                          receiptFetcher: receiptFetcher,
+                                                          purchaserInfoManager: purchaserInfoManager,
+                                                          backend: backend,
+                                                          identityManager: identityManager,
+                                                          receiptParser: receiptParser,
+                                                          deviceCache: deviceCache)
+        self.init(appUserID: appUserID,
+                  requestFetcher: fetcher,
+                  receiptFetcher: receiptFetcher,
+                  attributionFetcher: attributionFetcher,
+                  attributionPoster: attributionPoster,
+                  backend: backend,
+                  storeKitWrapper: storeKitWrapper,
+                  notificationCenter: NotificationCenter.default,
+                  systemInfo: systemInfo,
+                  offeringsFactory: offeringsFactory,
+                  deviceCache: deviceCache,
+                  identityManager: identityManager,
+                  subscriberAttributesManager: subscriberAttributesManager,
+                  operationDispatcher: operationDispatcher,
+                  introEligibilityCalculator: introCalculator,
+                  receiptParser: receiptParser,
+                  purchaserInfoManager: purchaserInfoManager,
+                  productsManager: productsManager,
+                  offeringsManager: offeringsManager,
+                  purchasesOrchestrator: purchasesOrchestrator)
+    }
+
+    @objc
+    public init(appUserID: String?,
```

Comment:
this one should not be `public` or `@objc`, it's solely for testing

Reply:
✅